free-programming-books-fr.md

```diff
@@ -161,7 +161,7 @@
 
 #### LaTeX
 
-* [LaTeX... pour le prof de maths !](http://math.univ-lyon1.fr/irem/IMG/pdf/LatexPourProfMaths3.pdf) - Arnaud Gazagnes (PDF)
+* [LaTeX... pour le prof de maths !](http://nolotv.free.fr/docs/LatexPourProfMaths.pdf) - Arnaud Gazagnes (PDF)
```

Comment:
Your link didn't work for me, but this link did:
http://math.univ-lyon1.fr/irem/IMG/pdf/LatexPourLeProfDeMaths.pdf

Could you verify?

Reply:
Your link works. I update the pull request.